src/routes/UserSearchContainer.js

```diff
@@ -17,6 +17,7 @@ import { UserSearch } from '../views';
 
 const INITIAL_RESULT_COUNT = 30;
 const RESULT_COUNT_INCREMENT = 30;
+const MAX_LIMIT = 2147483647;
```

Comment:
I am thinking about addind comment here in order to make it clear why this exact value was used.